content/doc/developer/plugin-governance/managing-permissions.adoc

```diff
@@ -0,0 +1,68 @@
+For all new repositories we tend to grant `Admin` permissions at the moment, and other plugin teams can get permissions elevated upon a request.
+
+How to request GitHub permissions?
+You will need to send a permission transfer request and to specify the permission recipient's GitHub account and Jenkins LDAP(aka Jenkins Jira) account there.
+
+* Request permissions change in Jenkins Jira
+** Project: `INFRA`, Component: `github`
+* Request permissions in the link:https://groups.google.com/forum/#!forum/jenkinsci-dev[:Jenkins developer mailing list].
+
+Permission change requests should be ideally done by one of the active plugin maintainers.
+Otherwise, an explicit approval will be required before the permission transfer happens.
+If there are multiple active maintainers in a plugin, Jenkins GitHub administrators will usually wait for a while to collect approvals from a number of them.
+
+=== Valid approvals by maintainers
+
+The following approvals are generally recognized as valid:
+
+* Confirmation in a Jenkins Jira ticket (permission request or other ticket)
+* Explicit approval in a GitHub pull request or issue
+** The reply should be sent from a GitHub account associated with a maintainer's Jenkins LDAP account
+* Confirmation in the link:https://groups.google.com/forum/#!forum/jenkinsci-dev[developer mailing list].
+** The reply should be sent from the email listed in the Jenkins LDAP account
+
+Other types of approval (emails from different addresses, email forwards by requesters, etc.) will be reviewed and verified by `jenkinsci` GitHub administrators.
+
+== Release Permissions
+
+Permissions to upload plugin releases are independent of GitHub push access and maintained in the link:https://github.com/jenkins-infra/repository-permissions-updater[Repository Permissions Updater] repository.
+
+To request upload permissions for a new maintainer:
+
+. If you have never done it before, 
+  you log in at least in once with your Jenkins account into the link:https://repo.jenkins-ci.org/webapp/#/home[Jenkins artifact repository].
+  Any modification to the permission files will be ineffective until then.
+. File a PR in the link:https://github.com/jenkins-infra/repository-permissions-updater[Repository Permissions Updater] for the specific plugin repository which needs the permission change.
+  Refer to the link:https://github.com/jenkins-infra/repository-permissions-updater/blob/master/README.md[Repository Permissions Updater README] for more detailed instructions. 
+  Once the permissions are updated, you'll be able to release your plugin.
+
+//TODO(oleg-nenashev): Add CoC violations and so on?
+
+By default we *DO NOT* remove upload permissions for current or past maintainers.
+It can be done based on the explicit approval of the previous maintainer or based on a discussion in the link:https://groups.google.com/forum/#!forum/jenkinsci-dev[Developer mailing list].
```

Comment:
is this needed? i.e. I've maintained a plugin for over a year, there's ~5 other people with commit / release access who haven't touched it in that time.
Shouldn't I just be able to remove them?

Reply:
Should or should not is a quite separate question. I am just trying to interpret the current governance model. @rodrigc has already created a JEP with a proposal to change it, and I am in favor of changes (team/company ownership, maintainer removal and so on). But it is rather a further change after the Governance board is reestablished

Reply:
I interpret this to be a description of default behavior in the project, not mandatory behavior by everyone that participates in the project.  Since you are a "previous maintainer" and a "current maintainer", I think you could announce to the dev list that you intend to remove specific people from the list, then proceed with the removal.  If they object, they can be added again.

Reply:
At least while I was involved in RPU, this wasn't that strict; maintainers could remove other maintainers by filing a PR and waiting a week or so (or getting explicit confirmation). I wanted to wait a bit to prevent some weird dispute from escalating. Obviously it's not good enough in the "vacation" case, but then there's a track of the previous maintainership status and could be reinstated. Obviously at that point it needs escalating/mediation anyway.